Assets/PlaygroundProject/PortalsSystem/Scripts/PortalLinks.cs

```diff
@@ -44,9 +44,9 @@ private void Awake()
         DontDestroyOnLoad(gameObject);
     }
 
-    [ReadOnly] [SerializeField] List<string> inputKeysList;
-    [ReadOnly] [SerializeField] List<Portal> portalsList;
-    [ReadOnly] [SerializeField] List<int> scenesList;
```

Comment:
Remove this [ReadOnly] is not recomendable in the actual moment. This is not a default function on Unity, so letting it here to remember to use this custom function in the future is a good idea.

Reply:
Done!